MatrixSDK/Crypto/KeyBackup/MXKeyBackup.m

```diff
@@ -1154,6 +1154,8 @@ - (MXKeyBackupVersionTrust *)trustForKeyBackupVersionFromCryptoQueue:(MXKeyBacku
                 }
                 else
                 {
+                    keyBackupVersionTrust.usable = YES;
```

Comment:
Only a month and a half later I've finally addressed this feedback, thanks @Anderas !

Reply:
Thanks for making the change. I see that we will now be adding a signature object even if the master key verification did not succeed, but it will be set as invalid (plus this copies js / android behaviour) so it sounds good to me